src/components/FeatureViews/ManagePayments/ManagePayments.js

```diff
@@ -0,0 +1,54 @@
+import React, {useCallback, useState} from "react";
+import MyPaymentsRoutes from "../../Routes/MyPaymentsRoutes";
+import BackgroundPaper from "../../OmouComponents/BackgroundPaper";
+import Typography from "@material-ui/core/Typography";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import Box from "@material-ui/core/Box";
+import {useHistory} from "react-router-dom";
+import ActiveInvoices from "./ActiveInvoices";
+
+function TabPanel(props) {
+	const {children, value, index, ...other} = props;
+
+	return (
+		<div
+			role="tabpanel"
+			hidden={value !== index}
+			id={`simple-tabpanel-${index}`}
+			aria-labelledby={`simple-tab-${index}`}
+			{...other}
+		>
+			{value === index && (
+				<Box p={3}>
+					<Typography>{children}</Typography>
+				</Box>
+			)}
+		</div>
+	);
+}
+
+export default function ManagePayments() {
+	const [value, setValue] = useState(0);
+	const history = useHistory();
+
+	const handleChange = useCallback((_, newTab) => {
+		setValue(newTab);
+		if (newTab === 0) history.push("/my-payments")
+		if (newTab === 1) history.push("/my-payments/history")
```

Comment:
Will there be more tabs in the future? If so, consider converting this to an object lookup

Reply:
not likely